### PR TITLE
Create sysmon_disabled_pua_protection_on_microsoft_defender.yml

### PR DESCRIPTION
**Pull request by austinsonger**
I don't know if this will work how I want it to.

```
[HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender]
"PUAProtection"=dword:00000000
```

### CODE REVIEW

File: rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+status: experimental
+date: 2021/08/04
+author: Austin Songer @austinsonger
+references:
+    - https://www.tenforums.com/tutorials/32236-enable-disable-microsoft-defender-pua-protection-windows-10-a.html
+tags:
+    - attack.defense_evasion
+logsource:
+    category: registry_event
+    product: windows
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml:13]: There is a tabulation \t after windows to remove

File: rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    category: registry_event
+    product: windows	
+detection:
+    selection:
+        EventType: SetValue
+        TargetObject|contains: 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender'
+        TargetObject|endswith:
+            - PUAProtection
+        Details: 'DWORD (0x00000000)'
+    condition: selection
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml:21]: There is a tabulation \t after selection to remove

File: rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+tags:
+    - attack.defense_evasion
+logsource:
+    category: registry_event
+    product: windows	
+detection:
+    selection:
+        EventType: SetValue
+        TargetObject|contains: 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender'
+        TargetObject|endswith:
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml:18]: `TargetObject|endswith: PUAProtection` works well as only 1 value

**Review by frack113 — CHANGES_REQUESTED** [review]
Just a fast review

File: rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    - https://www.tenforums.com/tutorials/32236-enable-disable-microsoft-defender-pua-protection-windows-10-a.html
+tags:
+    - attack.defense_evasion
+logsource:
+    category: registry_event
+    product: windows	
+detection:
+    selection:
+        EventType: SetValue
+        TargetObject|contains: 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows Defender'
```
Comment (frack113) [inline rules/windows/registry_event/sysmon_disabled_pua_protection_on_microsoft_defender.yml:17]: on sysmon sould be `HKLM\SOFTWARE\Policies\Microsoft\Windows Defender\PUAProtection`

**Review by frack113 — CHANGES_REQUESTED** [review]
need to test on a VN but sure for HKLM

**Comment by frack113** [comment]
Hi,
Add tags `attack.t1562.001` https://attack.mitre.org/techniques/T1562/001/ Impair Defenses: Disable or Modify Tools 
Thanks

**Review by frack113 — APPROVED** [review]

**Review by StevenDias33 — APPROVED** [review]

**Pull request merged**